examples/ubuntu-ui-toolkit-gallery/manifest.json.in

```diff
@@ -1,15 +1,15 @@
 {
     "architecture": "all",
     "description": "Component showcase gallery of the Ubuntu user interface toolkit (UITK)",
-    "framework": "ubuntu-sdk-15.04.3",
+    "framework": "ubuntu-sdk-16.04",
     "hooks": {
         "ubuntu-ui-toolkit-gallery": {
             "apparmor": "ubuntu-ui-toolkit-gallery.apparmor",
             "desktop": "ubuntu-ui-toolkit-gallery.desktop"
         }
     },
-    "maintainer": "Zoltán Balogh <zoltan.balogh@ubuntu.com>",
-    "name": "com.ubuntu.developer.bzoltan.ubuntu-ui-toolkit-gallery",
+    "maintainer": "UBports <dev@ubports.com>",
+    "name": "com.ubports.ubuntu-ui-toolkit-gallery",
```

Comment:
If we're going to be renaming the package anyway, maybe we could make it simpler and use the newer style of naming instead? I would suggest having it be `uitk-gallery.ubports` for a nice simple name.

Reply:
need to find out how to generate the pot file also